### PR TITLE
ROE-1039 Allow both choices of who is verifying to pour into id check…

### PR DESCRIPTION
**Pull request by mattch1**
… section

### JIRA link

https://companieshouse.atlassian.net/browse/ROE-1039

### Change description

Depending on what is selected the change link urls will vary in this section.

### Work checklist

- [x] Tests added where applicable
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

File: src/config/index.ts [hunk 1]
```diff
@@ -114,12 +114,28 @@ export const ENTITY_CHANGE_LEGAL_FORM = ENTITY_URL + LEGAL_FORM;
 export const ENTITY_CHANGE_GOVERNING_LAW = ENTITY_URL + LAW_GOVERNED;
 export const ENTITY_CHANGE_PUBLIC_REGISTER = ENTITY_URL + "#is_on_register_in_country_formed_in";
 
+const IDENTITY_DATE = "#identity_date-day";
+const IDENTITY_NAME = "#name";
```
Comment (mouhajer-ch) [inline src/config/index.ts:118]: Call it name, it is needed in other fields

File: views/includes/check-your-answers/identity-checks.html [hunk 2]
```diff
@@ -36,6 +36,63 @@
   {% endif %}
 {% endset %}
 
+{% set dateChangeLink %}
```
[the remark this answers is not in the document]
Comment (mattch1) [reply views/includes/check-your-answers/identity-checks.html:39]: We should probably leave it as it is.

File: src/config/index.ts (hunk 1 above)
Comment (mattch1) [reply src/config/index.ts]: Updated .

File: views/includes/check-your-answers/identity-checks.html (hunk 2 above)
[the remark this answers is not in the document]
Comment (mouhajer-ch) [reply views/includes/check-your-answers/identity-checks.html:39]: I will have a look

[the remark this answers is not in the document]
Comment (mouhajer-ch) [reply views/includes/check-your-answers/identity-checks.html:39]: Tested locally and it works. 
```
{% if appData.who_is_registering == "agent" %}
  {% set whoIsCompletingTheRegistration = "The UK agent" %}
  {% set dateChangeLink = OE_CONFIGS.DUE_DILIGENCE_CHANGE_IDENTITY_DATE %}
  {% set nameChangeLink = OE_CONFIGS.DUE_DILIGENCE_CHANGE_NAME%}
  {% set addressChangeLink = OE_CONFIGS.DUE_DILIGENCE_CHANGE_IDENTITY_ADDRESS%}
  {% set emailChangeLink = OE_CONFIGS.DUE_DILIGENCE_CHANGE_EMAIL%}
  {% set supervisoryChangeLink = OE_CONFIGS.DUE_DILIGENCE_CHANGE_SUPERVISORY_NAME%}
  {% set amlChangeLink = OE_CONFIGS.DUE_DILIGENCE_CHANGE_AML_NUMBER%}
  {% set partnerChangeLink = OE_CONFIGS.DUE_DILIGENCE_CHANGE_PARTNER_NAME%}
{% else %}
  {% set whoIsCompletingTheRegistration = "Someone from the overseas entity" %}
  {% set dateChangeLink = OE_CONFIGS.OVERSEAS_ENTITY_DUE_DILIGENCE_CHANGE_IDENTITY_DATE %}
  {% set nameChangeLink = OE_CONFIGS.OVERSEAS_ENTITY_DUE_DILIGENCE_CHANGE_NAME%}
  {% set addressChangeLink = OE_CONFIGS.OVERSEAS_ENTITY_DUE_DILIGENCE_CHANGE_IDENTITY_ADDRESS %}
  {% set emailChangeLink = OE_CONFIGS.OVERSEAS_ENTITY_DUE_DILIGENCE_CHANGE_EMAIL%}
  {% set supervisoryChangeLink = OE_CONFIGS.OVERSEAS_ENTITY_DUE_DILIGENCE_CHANGE_SUPERVISORY_NAME%}
  {% set amlChangeLink = OE_CONFIGS.OVERSEAS_ENTITY_DUE_DILIGENCE_CHANGE_AML_NUMBER%}
  {% set partnerChangeLink = OE_CONFIGS.OVERSEAS_ENTITY_DUE_DILIGENCE_CHANGE_PARTNER_NAME%}
{% endif %}
```

[the remark this answers is not in the document]
Comment (mattch1) [reply views/includes/check-your-answers/identity-checks.html]: Ok added the update to the PR and fixed an issue with the email url for oe due diligence.

**Review by mouhajer-ch — APPROVED** [review]

**Pull request merged**